.github/workflows/pages.yaml

```diff
@@ -0,0 +1,56 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy static content to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["master"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
Where is this? I can't seem to find it

Reply:
Good question!

I copy-pasted this `.github/workflows/pages.yaml` file from an example you get when you try to add one of these documentation-building actions.  The comment was from the example.  I'm not sure how this works.  It is also possible that it doesn't show up until you merge the action into `master`.

I've never used the "deployment" or "environments" feature from github actions, so we'll have to play around with this and try to figure out if there is anything else we can do with it.  For now at least it appears required for building documentation with github actions and deploying to github.io.

I'll go ahead and merge it for now, but if we realize this `workflow_dispatch` key doesn't work or something later, let's remove it.